R/util.r

```diff
@@ -153,21 +153,35 @@ find_first_existing <- function(path, ...) {
 #'
 #' @param path Relative path
 #' @param base Base path
+#' @param windows Whether the operating system is Windows. Default value is to
+#'   check the user's system information.
 #' @export
 #' @examples
 #' rel_path("a/b", base = "here")
 #' rel_path("/a/b", base = "here")
-rel_path <- function(path, base = ".") {
+rel_path <- function(path, base = ".", windows = on_windows()) {
   if (is_absolute_path(path)) {
     path
   } else {
     if (base != ".") {
       path <- file.path(base, path)
     }
-    normalizePath(path, mustWork = FALSE)
+    # normalizePath() on Windows expands to absolute paths,
+    # so strip normalized base from normalized path
+    if (windows) {
+      parent_full <- normalizePath(".", mustWork = FALSE, winslash = "/")
+      path_full <- normalizePath(path, mustWork = FALSE, winslash = "/")
+      gsub(paste0(parent_full, "/"), "", path_full)
```

Comment:
This should probably be `fixed = TRUE` for safety

Reply:
done!